tests/test_post.py

```diff
@@ -107,6 +106,311 @@ def plastic_solve(mapdl):
     mapdl.mute = False
 
 
+@pytest.fixture(scope="module")
+def contact_solve(mapdl):
+    mapdl.mute = True
```

Comment:
Good work with mute here. `non_interactive` would have worked as well.

Reply:
Thank you :)